### PR TITLE
Add option to use IdToken instead of AccessToken

### PR DESCRIPTION
**Pull request by sschadwick**
nuxt-community/auth-module#119

### CODE REVIEW

File: docs/schemes/oauth2.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ auth: {
       token_type: 'Bearer',
       redirect_uri: undefined,
       client_id: 'SET_ME',
+      id_token:  true
```
Comment (pi0) [inline docs/schemes/oauth2.md]: Default shouldn't be `false`?

**Comment by sschadwick** [comment]
You're absolutely right, I've updated the commit. I like the idea of using `tokenKey` as the key but it would involve string matching, which is never my first choice. However, if that's how you want to proceed, how does this sound: `tokenKey: 'access_token'` and `tokenKey: 'id_token'`?

**Comment by sschadwick** [comment]
Okay updated again. I used your suggestion but changed `tokenKey` to `token_key` to remain consistent with the other config options. Also added `access_token` as the default.

**Comment by pi0** [comment]
Thanks for your contribution!

**Pull request merged**